<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lNum0">
     <property name="text">
      <string>First Number:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="sbNum0">
     <property name="minimum">
      <number>-1000000000</number>
     </property>
     <property name="maximum">
      <number>1000000000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="bNum0">
     <property name="text">
      <string>Binary Representation</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lNum1">
     <property name="text">
      <string>Second Number:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="sbNum1">
     <property name="minimum">
      <number>-1000000000</number>
     </property>
     <property name="maximum">
      <number>1000000000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="bNum1">
     <property name="text">
      <string>Binary Representation</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lOpr">
     <property name="text">
      <string>Operation</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cbOpr"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pbCal">
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lRes">
     <property name="text">
      <string>Results:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="sbNumRes">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <number>-1000000000</number>
     </property>
     <property name="maximum">
      <number>1000000000</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="lNumRes">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
